changelog/8.8.2.asciidoc

```diff
@@ -0,0 +1,48 @@
+// begin 8.8.2 relnotes
+
+[[release-notes-8.8.2]]
+==  8.8.2
+
+Review important information about the  8.8.2 release.
+
+[discrete]
+[[security-updates-8.8.2]]
+=== Security updates
+
+
+elastic-agent::
+
+* Increase Go Version To 1.19.10. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2846[#https://github.com/elastic/elastic-agent/pull/2846] 
+
+
+
+
+
+
+
+
+
+
+[discrete]
+[[enhancements-8.8.2]]
+=== Enhancements
+
+
+elastic-agent::
+
+* Log Start And Stop Operations From Service Runtime At Info Level. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2879[#https://github.com/elastic/elastic-agent/pull/2879] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/2864[#https://github.com/elastic/elastic-agent/issues/2864]
```

Comment:
```suggestion
* Log start and stop operations from service runtime at `INFO` rather than `DEBUG` level. {elastic-agent-pull}https://github.com/elastic/elastic-agent/pull/2879[#https://github.com/elastic/elastic-agent/pull/2879] {elastic-agent-issue}https://github.com/elastic/elastic-agent/issues/2864[#https://github.com/elastic/elastic-agent/issues/2864]
```